heartbeat/ipsec

```diff
@@ -150,10 +150,14 @@ ipsec_monitor() {
 
 	ipsec status | grep "$OCF_RESKEY_tunnel" | grep -q unoriented
 	state=$?
+
+	# TODO  I think we need also to check if output of ipsec status is empty!
+	#       Then the tunnel is not running.
```

Comment:
If it is empty it wont match unoriented.

Reply:
Yes but if it is empty the check reports that the tunnel is active which is not true.

Reply:
Oh.